src/objects/entities/Ben3.ts

```diff
@@ -102,6 +102,8 @@ class Ben3 extends Entity {
 
   public headScale = HEAD_SCALE_MIN;
 
+  private isJumping = false;
```

Comment:
`isJumping` feels more like a state variable that indicates if the Ben3 is jumping

thiis is more like a `hasUsedJumpSinceTouchingGround` maybe?

i think traditionally "Coyote Jumps" (or "Coyote Time") is done with a timer in addition to what you have here